README.md

```diff
@@ -1,44 +1,109 @@
 
-To get the latest version, run this command:
+> ℹ️ **Please note:** Using the APIs requires an active Calibre account (in trial or on a paid plan).
 
-```
+## 🖇 Table of Contents
+
+1. [Installation](#-installation)
+2. [Authentication](#-authentication)
+3. [Features](#-features)
+4. [Usage](#-usage)
+5. [Package Exports](#-package-exports)
+6. [Script Examples](#-script-examples)
+7. [Resources](#-resources)
+
+## 📥 Installation
+
+To install the latest version globally, run this command:
+
+```bash
 npm install -g calibre
 ```
 
-**Other installation options**
-
-* [Install a standalone binary](https://calibreapp.com/docs/api/binaries) Mac, Windows or Linux.
-* Run the command directly with npx: `npx calibre --help`
+or save it directly to your project:
 
-## Features
+```bash
+npm install calibre --save
+```
 
-* **Manage your performance system with a single command**: `calibre`.
-* **Single page tests**. Create one-off, sharable Calibre tests for fast feedback: `calibre test create <url> --location=Frankfurt --device=iPhone6`
-* **Deployments**. Track performance, best practices, accessibility and SEO between deploys
-* **CI/CD/Automation**. Every command has a --json flag for machine readable output: `calibre test <url> --location=Sydney --json`
-* **Scriptable**. All API methods are exported to the Calibre NPM package
+The `calibre` package is built to support for **all current Node.js LTS releases**.
 
-## Authentication
+## 🔐 Authentication
 
-Authentication is handled via an environment variable: `CALIBRE_API_TOKEN`. Create your token via the team API tokens page.
+To authenticate to Calibre and use the APIs, **you will need to create or use an existing API Token**. Learn about the [types of API Tokens and how to create them here](https://calibreapp.com/docs/account-and-billing/manage-api-tokens).
 
-## Usage
+Once you have an API Token, you can [store it locally](https://calibreapp.com/docs/automation/tokens#store-a-token-locally):
 
+```bash
+calibre token set <your token>
 ```
+
+Or [as an environment variable](https://calibreapp.com/docs/automation/tokens#store-a-token-as-an-environment-variable).
+
+## 💡 Features
+
+- **Manage performance with a single command**. Retrieve all speed data and manage the test environment and settings.
+- **Run [Single Page Tests](https://calibreapp.com/docs/automation/single-page-tests)**. Create one-off tests privately or share them with your team.
+- **Monitor speed across your release process**. Track performance between deploys and mark them on your charts.
+- **Use Calibre within CI/CD**. Every command has a `--json` flag for machine-readable output.
+- **Script any action**. We export all API methods to the Calibre `npm` package..
+
+## 🛠 Usage
+To see a full list of available commands, subcommands and options run:
+
+```bash
 $ calibre --help
+$ calibre <command> --help
+```
+
+or [see all commands](commands.md) in this repository.
```

Comment:
@benschwarz here's the upcoming link to the commands markdown file, feel free to amend the name.

Reply:
@thefoxis Thanks! I'll update once I've dropped in the markdown render